### ISSUE TITLE
The result of MS-GF+

### ISSUE THREAD

**Issue by Golden-proteogenomics** [MSGFPlus/msgfplus#96]
I want to konw our software MG-GF+ would generate files for complement submission in ProteomeXchange?

**Comment by alchemistmatt** [comment]
Yes, MS-GF+ creates .mzid files, and those files are compatible with ProteomeXchange
